README.md

```diff
@@ -685,6 +685,15 @@ Specifies an option to specify whether this source should explicitly bypass any
 Requires at least Chocolatey v0.10.4.
 Defaults to `false`.
 
+#### `admin_only`
+
+(**Property**: This parameter represents a concrete state on the target system.)
+
+Option to specify whether this source should visible to administrators only.
+
+Requires at least Chocolatey v0.10.8 to set and at least Chocolatey Business Edition v1.12.2 to take effect.
```

Comment:
"Chocolatey for Business (C4B)" not "Chocolatey Business Edition"

I would reorder this a bit: "Requires Chocolatey for Business (C4B) v1.12.2+ and at least Chocolatey v0.10.8 for the setting to be respected." That way the commercial expense requirement is noted first.

Reply:
Fixed!